README.md

```diff
@@ -1,20 +1,44 @@
-# Beekeeper
-
 # Overview
-Beekeeper is a service that schedules orphaned paths for cleanup.
+
+Beekeeper is a service that schedules orphaned paths for deletion.
+
+The original inspiration for a data deletion tool came from another of our open source projects called [Circus Train](https://github.com/HotelsDotCom/circus-train). At a high level, Circus Train replicates Hive datasets. The datasets are copied as immutable snapshots to ensure strong consistency and snapshot isolation, only pointing the replicated Hive Metastore to the new snapshot on successful completion. This process leaves behind snapshots of data which are now unreferenced by the Hive Metastore, so Circus Train includes a Housekeeping module to delete these files later.
+
+Circus Train’s Housekeeping module has been re-written and updated to create this service.
+
+## Start using
+
+To deploy Beekeeper in AWS, see the [terraform repo](https://github.com/ExpediaGroup/beekeeper-terraform). 
+
+Docker images can be found in Expedia Group's [dockerhub](https://hub.docker.com/u/expediagroup). 
```

Comment:
can we make this URL specific to Beekeeper. The main URL has other docker images too.

perhaps, https://hub.docker.com/search/?q=expediagroup%2Fbeekeeper

Reply:
looks good, will make that change